client/apollo/react/src/ContentItemMono/ContentItemMonoCommon.tsx

```diff
@@ -54,7 +54,7 @@ export const getContentItemCoreProps = ({
       title,
       primarySubtitle: subtitle1,
       subtitle: subtitle2,
```

Comment:
On ne pourrait pas uniformiser les noms ?
```suggestion
      primarySubtitle,
      subtitle,
```

Reply:
ah comment ?

Reply:
en changeant les props mais bon ce n'est pas urgent, on peut laisser comme ça pour le moment